src/pages/debug-data.php

```diff
@@ -38,8 +38,7 @@
 			}
 			?>
 			<div id="system-information-english-copy-wrapper" style="display: none;">
-					<textarea id="system-information-english-copy-field" class="widefat" rows="10">`
-						<?php
+					<textarea id="system-information-english-copy-field" class="widefat" rows="10">`<?php
```

Comment:
The PHP opening tag should be on the next line down to both conform with the coding standards, and also be properly parseable by the WordPress.org forums as codetags. Removing the indentations before it on a newline should have the same desired effect.